### PR TITLE
[Update] up new version for simple-maven-project Test 

### PR DESCRIPTION
**Pull request by Jzow**
I think we should update the new version for testing. At present, we are using CoreNLP. I will add some interesting tests later, including all languages. Thanks

### CODE REVIEW

**Comment by AngledLuffa** [comment]
Would you undo the whitespace changes and redirect the change to our dev
branch?

**Comment by Jzow** [comment]
@AngledLuffa you mean the PR to dev branch?

**Comment by AngledLuffa** [comment]
Yes, and please adjust the whitespace so the PR is actually readable

**Review by Jzow — COMMENTED** [review]
@AngledLuffa Like this?

**Comment by Jzow** [comment]
I don't think there is a whitespace right now

File: examples/sample-maven-project/src/main/java/edu/stanford/nlp/StanfordCoreNLPChineseTestApp.java [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-{
-    public static void main(String[] args) throws IOException, ClassNotFoundException
-    {
-        String[] chineseArgs = new String[]{"-props","StanfordCoreNLP-chinese.properties",
-                                              "-annotators","tokenize,ssplit,pos,lemma,ner,parse,coref,kbp,entitylink",
-                                              "-file", "sample-chinese.txt", "-outputFormat", "text"};
-        StanfordCoreNLP.main(chineseArgs);
+    public static void main(String[] args) {
+        Properties properties = new Properties();
+        properties.setProperty("annotators", "tokenize,ssplit,pos,lemma,ner,parse,coref,kbp,entitylink");
```
Comment (AngledLuffa) [inline examples/sample-maven-project/src/main/java/edu/stanford/nlp/StanfordCoreNLPChineseTestApp.java:14]: Chinese properties for the Chinese test app?

File: examples/sample-maven-project/src/main/java/edu/stanford/nlp/StanfordCoreNLPSpanishTestApp.java [hunk 2]
```diff
@@ -2,19 +2,24 @@
...
-{
-    public static void main(String[] args) throws IOException, ClassNotFoundException
-    {
-        String[] chineseArgs = new String[]{"-props","StanfordCoreNLP-spanish.properties",
-                                              "-annotators","tokenize,ssplit,pos,lemma,ner,parse,depparse,kbp",
-                                              "-file", "sample-spanish.txt", "-outputFormat", "text"};
-        StanfordCoreNLP.main(chineseArgs);
+public class StanfordCoreNLPSpanishTestApp {
+
+    public static void main(String[] args) throws IOException, ClassNotFoundException {
```
Comment (AngledLuffa) [inline examples/sample-maven-project/src/main/java/edu/stanford/nlp/StanfordCoreNLPSpanishTestApp.java:14]: Again, I would expect the Spanish properties to be needed here

File: examples/sample-maven-project/pom.xml [hunk 3]
```diff
@@ -1,83 +1,89 @@
...
+    <groupId>edu.stanford.nlp</groupId>
+    <artifactId>stanford-corenlp-test-app</artifactId>
+    <packaging>jar</packaging>
+    <version>1.0-SNAPSHOT</version>
+    <name>stanford-corenlp-test-app</name>
+    <url>http://maven.apache.org</url>
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <stanford.corenlp.version>4.2.2</stanford.corenlp.version>
```
Comment (AngledLuffa) [inline examples/sample-maven-project/pom.xml:13]: 4.4.0 now

File: examples/sample-maven-project/pom.xml [hunk 4]
```diff
@@ -1,83 +1,89 @@
...
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>${stanford.corenlp.version}</version>
+            <classifier>models-french</classifier>
+        </dependency>
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>4.2.1</version>
```
Comment (AngledLuffa) [inline examples/sample-maven-project/pom.xml:79]: ${stanford.corenlp.version}   ?

**Comment by AngledLuffa** [comment]
I left a few small notes here.  (The whitespace still shows up, but I found
the "hide whitespace" button)

**Comment by Jzow** [comment]
@AngledLuffa Please check, Thanks

File: examples/sample-maven-project/src/main/java/edu/stanford/nlp/StanfordCoreNLPChineseTestApp.java (hunk 1 above)
Comment (AngledLuffa) [reply examples/sample-maven-project/src/main/java/edu/stanford/nlp/StanfordCoreNLPChineseTestApp.java:16]: I still don't think this is it... the Chinese properties are not being loaded and the "annotators" field is overwritten right away.  What's the purpose for this update?  Is the original version not working?

File: examples/sample-maven-project/pom.xml [hunk 5]
```diff
@@ -1,83 +1,89 @@
...
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <stanford.corenlp.version>4.4.0</stanford.corenlp.version>
+        <junit.version>4.13.1</junit.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
```
Comment (AngledLuffa) [inline examples/sample-maven-project/pom.xml:20]: I like the idea of separating out these variables.  If you would make just this change into a separate pull request, we'll merge it.  Maybe a <name> field which still reflects the corenlp name, though.

**Comment by Jzow** [comment]
If don't merge, I will close the PR

**Comment by AngledLuffa** [comment]
> If don't merge, I will close the PR

Happy to, but I don't think the issues are resolved:

- The whitespace changes in pom.xml are extensive and make it hard to read the actual change.  If I go to "Files Changed", all of examples/sample-maven-project/pom.xml is edited, whereas most of it is whitespace changes.
- The Chinese test app has `properties.setProperty("annotators", "StanfordCoreNLP-chinese.properties");` which is overwritten by the next statement anyway.  I believe it needs to be `setProperty("props", ...)`

**Comment by Jzow** [comment]
sorry I only saw it now, but I don't think it has solved the actual problem, so I closed it
